wdns.spec

```diff
@@ -11,8 +11,8 @@ BuildRequires:  gcc
 #Requires:       
 
 %description
-wdns is a low-level DNS library. It contains a fast DNS message parser
```

Comment:
@djw1149 merged these changes into this branch, but sure..

Reply:
I don't think I put the version of wdns.spec into this branch.
On 4/7, I corrected a typo in Allan's 1/15 commit of wdns.spec.
The correct text for it includes: "wdns is a lower-level DNS library. It contains a fast DNS message parser".
The wdns.spec in the https-dns branch *today* predates Allan's changes.

Reply:
@djw1149 - The .spec changes came in via the following merge commit: https://github.com/farsightsec/wdns/commit/3133ece1741e9aa6d7c71dd90560182f050c6241

In any case, my intent with the comment was not to blame (git or otherwise), but to note that this should be left out when merging this PR. In general, I would prefer packaging updates to be separate from the feature work, with a possible exception if the feature work changes dependencies.